tools/pebblescout.md

```diff
@@ -2,60 +2,81 @@
-provides a .tsv file containing the SubjectID of all SRA runs where the
-query sequence is found, along with a RawScore, %coverage, PBSscore,
-Biosample, and title of the study the BioSample is found in. A web
-version of the table will be available to view in the browser.
+The primary output is a tab-separated values (.tsv) file containing a 
+list of sequencing runs where your query was found. Rows are runs, 
+columns are: 
+-  SubjectID, the accession number of the run
+-  Raw Score, the sum of kmer scores for all kmers considered from the
+query with a match to the subject 
+-  %coverage, the percentage of all kmers considered from the query with
+match to the subject 
+-  PBSscore, the normalized score 
+-  BioSample, the accession number of the BioSample from which
+the run is taken
+-  Title of the study in which the BioSample is found
 
 ### 1. Navigate to [PebbleScout](https://pebblescout.ncbi.nlm.nih.gov/)
 
 ### 2. Input your query
+Copy and paste your query sequence, or the GenBank accession number of
+your query sequence, into the input window, or upload a FASTA file 
+containing your query sequence.
 
-Copy the sequence for your obelisk, and paste it into the input window.
-
-### 2. Select Database
+### 3. Select Database
 
-In "Select Database and Report", select "Metagenomic" database - this is
-most relevant for our purposes, but feel free to explore other databases
-as well.
+In "Select Database and Report", select your database of choice. This is 
+the collection PebbleScout will search. 
 
-### 3. Adjust parameters (optional)
+### 4. Adjust parameters if needed
 
-For your query search, you can adjust: - Masking threshold (k-mers
-sampled from more subjects than this amount will be ignored) - Score
-constant (Effects the calculation of RawScores, learn more
-[here](https://pebblescout.ncbi.nlm.nih.gov/?view=doc#s-3)) - Max number
-of subjects per query (the maximum number of outputs reported)
+While default paramaters suffice for general purposes, you may adjust: 
+- Masking threshold (A lower masking threshold means a higher number of
+common k-mers will be automatically filtered out of results)
+- Score constant (Affects calculation of RawScore, learn more
+[here](https://pebblescout.ncbi.nlm.nih.gov/?view=doc#s-3))
+- Max number of subjects per query (the maximum number of outputs 
+returned)
 
 ![PebbleScout example inputs](img/PebbleScout/PebbleScoutInput.png)
 
-### 4. Run & view or download data
+### 5. Run & view or download data
 
 Click "View" at the bottom of the page to view the results in browser,
 click "Download" to download the results as a .tsv file. 
 ![PebbleScout Example Output](img/PebbleScout/PebbleScoutOutput.png)
 
+
+### Interpreting Results
+
+PebbleScout measures output confidence by RawScore and PBSscore. Kmers are 
+scored by dividing the score constant C, specified for the database, by
+the sum of C and the number of subjects the Kmer, K, is found in within 
+the database. 
```

Comment:
Can you add the math equation for this calculation too here?

Something like this

$e_i^k = \sum_{j=1}^{20} P_{ij} \cdot c_j^k$ [IUPRED2 example](https://github.com/ababaian/VIRUSxDISCVRY/pull/83)